lua/toggleterm/ui.lua

```diff
@@ -316,11 +333,18 @@ end
 ---Resize a split window
 ---@param term Terminal
 ---@param size number
-function M.resize_split(term, size)
-  if term:is_split() then
-    size = M._resolve_size(size or M.get_size(), term)
-    vim.cmd(split_commands[term.direction].resize .. " " .. size)
+---@param resurrect_only boolean
+function M.resize_split(term, size, resurrect_only)
```

Comment:
@zeertzjq so what I'm suggesting is
```lua
function M.resize_split(opts)
 local term, size, save_size = opts.term, opts.size, opts.save_size
 --- ... other lines in between then
 -- sizes have not been saved on session resurrect
 if save_size and persist_size and persistent[term.direction] then
  return
 end
 size = M._resolve_size(M.get_size(size, term.direction), term)
 if save_size then
   M.save_direction_size(size, term.direction)
 end
end
```

So the two places where you would set `save_size` are when resurrecting and when resizing split and so you don't need to call `save_direction_size` outside this module? Will that work or am I missing something